index.js

```diff
@@ -22,6 +39,12 @@ function checkAltText(ast, file) {
       if (node.alt && altText(node.alt)) {
         file.message(altText(node.alt), node);
       }
+      if (imageIsLink && hasAltText) {
```

Comment:
`hasAltText` should calculated per node and not if any node in a file has one alt text